src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/redundantthis/InputRedundantThisCatchVariables.java

```diff
@@ -0,0 +1,40 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.redundantthis;
+
+public class InputRedundantThisCatchVariables extends Thread {
```

Comment:
` extends Thread ` why it is here?

Please try to not include into test input anything that is not important for check

Reply:
Done.